### ISSUE TITLE
Feature suggestion: Support for better representation of inline models

### ISSUE THREAD

**Issue by RuchitMicro** [unfoldadmin/django-unfold#594]
Currently, the inline models (TabularInline and StackedInline) are shown below the main form of the model.
We need a UI that adds tabs for all the different Inline Models registered under a single model.

**Comment by lukasvinclav** [comment]
This was released literally few minutes ago :-)

**Comment by RuchitMicro** [comment]
OMG thank you so much.

**Comment by RuchitMicro** [comment]
I can't see it in the commit tree, could you please link the PR. Thanks once again.

**Comment by RuchitMicro** [comment]
I found it 
it's implemented in version 0.31.0

here is the code for future wanderers

```
class GenericStackedAdmin(TabularInline):
    extra   = 1
    tab     = True #<----- Add this flag and tabs will show up for your inline models
```
